notes/release-train-issue-template.md

```diff
@@ -0,0 +1,72 @@
+Release Train Issue Template for Akka HTTP
+
+(Liberally copied and adopted from Scala itself https://github.com/scala/scala-dev/blob/b11cd2e4a4431de7867db6b39362bea8fa6650e7/notes/releases/template.md)
+
+For every Akka Http release, make a copy of this file named after the release, and expand the variables.
+Ideally replacing variables could become a script you can run on your local machine.
+
+Variables to be expanded in this template:
+- AKKA_HTTP_VER=??? (currently not used)
+
+Key links:
+  - akka/akka-http milestone: https://github.com/akka/akka-http/milestone/?
+
+### N weeks before the release
+- [ ] Check that open PRs and issues assigned to the milestone are reasonable
+- [ ] Triage tickets that should be ready for this release, add "pick next" label and release milestone
+- [ ] Triage open PRs and apply "pick next" label and maybe add to release milestone. Some PRs might be explicitly scheduled for this release, others might be ready enough to bring them over the finish line 
+Wind down PR queue. There has to be enough time after the last (non-trivial) PR is merged and the next phase. The core of the eco-system needs time to prepare for the final!
+- [ ] Estimate expected release date
+
+### ~ 1 week before the release
+- [ ] Decide on planned release date
+- [ ] Notify depending projects (notably Play + cinnamon) about upcoming release
+- [ ] Create a new milestone for the next version at https://github.com/akka/akka-http/milestones
+
+### 1 day before the release
+- [ ] Make sure all important / big PRs have been merged by now
+- [ ] Check that latest nightly (once we have that) / master still works with depending projects (notably Play + cinnamon)
+- [ ] Communicate that a release is about to be released in [Gitter Akka Dev Channel](https://gitter.im/akka/dev), so that no new Pull Requests are merged
+
+### Preparing release notes in the documentation / announcement
+
+- [ ] If this is a new minor (not patch) release, rename the 'akka-http-x.x-stable' reporting project in [WhiteSource](https://saas.whitesourcesoftware.com/) accordingly
+- [ ] Add a release notes entry in docs/src/main/paradox/release-notes/. As a helper run `scripts/commits-for-release-notes.sh <last-version-tag>` which will output a list of commits grouped by submodule, and the closed issues for this milestone.
+- [ ] Create a news item draft on https://github.com/akka/akka.github.com, using the milestones and `scripts/authors.scala previousVersion thisVersion`
+- [ ] In the same PR update Akka HTTP dependency for akka.io website: https://github.com/akka/akka.github.com/blob/master/_config.yml
+- [ ] Move all unclosed issues there and close the version you're releasing
```

Comment:
This needs to move to another section, and either be under the 'create new milestone' step or be adapted so it's clear this is about moving issues to the new milestone ;)

Reply:
Done.